### ISSUE TITLE
Install documentation fail

### ISSUE THREAD

**Issue by oguilb** [mattermost/react-native-network-client#103]
npm install @mattermost/react-native-network-client --save
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@mattermost%2freact-native-network-client - Not found
npm ERR! 404 
npm ERR! 404  '@mattermost/react-native-network-client@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

**Comment by enahum** [comment]
Yes @oguilb so sorry for that, we haven't publish the library to nom yet, in the meantime you can add the package as a github reference in your package.json file

**Comment by oguilb** [comment]
Thank's but It not work also.
I have append this in my package.json but npm install fail 
"react-native-network-client": "github:mattermost/react-native-network-client",

If I replace "mattermost" by a fork, it work

**Comment by enahum** [comment]
Your package.json should have it like this `"@mattermost/react-native-network-client": "github:mattermost/react-native-network-client"`

**Comment by oguilb** [comment]
I have tested but fail also

npm install 
The authenticity of host 'github.com (140.82.121.4)' can't be established.e
ECDSA key fingerprint is SHA256:p2QAMXNIC1TJYWeIOttrVc98/R1BUFWu3/LiyKgUfQM.

and freze here

**Comment by enahum** [comment]
that looks like your host does not have ssh keys, perhaps change it with `git+https://github.com/mattermost/react-native-networl-client.git` and see if that works
